query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/persist/CreateLdapConfiguration.java

```diff
@@ -145,6 +160,24 @@ public void validate() {
     if (containsErrorMsgs()) {
       return;
     }
+
+    List<StringField> claimArgs =
+        claimMappings
```

Comment:
We should be validating the claims passed in through the LdapConfiguration argument, not creating an empty claims mapping field. https://github.com/connexta/admin-console/blob/17433eefd7d08af2e1a42bb6130411099d421a6b/query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/config/LdapConfigurationField.java#L77